testing/recordings/TestOrthographicProjectionCommand.log

```diff
@@ -0,0 +1,4 @@
+# StreamVersion 1.2
+ExposeEvent 0 298 0 0 0 0 0
+RenderEvent 0 298 0 0 0 0 0
```

Comment:
still not an interaction test, not needed.

Reply:
I got an error to say that this file was missing when running the CI.
How do I tell the system that it is not an interaction test?

Reply:
In the latest CI

```
83/318 Test  #81: f3d::TestOrthographicProjectionCommand .............................***Failed    0.23 sec
Interaction record file to play does not exist /__w/f3d/f3d/source/testing/recordings/TestOrthographicProjectionCommand.log
```
https://github.com/f3d-app/f3d/actions/runs/8314790526/job/22752385391

Reply:
Nevermind, I saw the `INTERACTION` in the test case